extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/AdditionalHttpServerMetricsTagsProvider.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.micrometer.runtime;
+
+import io.micrometer.core.instrument.Tags;
+import io.vertx.core.http.HttpServerRequest;
+
+/**
+ * Allows code to add additional Micrometer {@link Tags} to the metrics collected for completed HTTP server requests.
+ * <p>
+ * The implementations of this interface are meant to be registered via CDI beans.
+ */
+public interface AdditionalHttpServerMetricsTagsProvider {
+
+    Tags configure(Context context);
+
+    interface Context {
```

Comment:
The reason for using this additional interface is that it allows us to introduce new items items to the API without breaking anything (as opposed to adding a new parameter to `configure`)